packages/utils/typescript/lib/compile.js

```diff
@@ -5,7 +5,7 @@ const getConfigPath = require('./utils/get-config-path');
 
 module.exports = async (srcDir, { watch = false, configOptions = {} } = {}) => {
   // TODO: Use the Strapi debug logger instead or don't log at all
-  console.log(`Starting the compilation for TypeScript files in ${srcDir}`);
+  process.stderr.write(`Starting the compilation for TypeScript files in ${srcDir}\n`);
```

Comment:
Based on the TODO comment above, I think it would make more sense to remove the debug message totally. It doesn't provide much value anyway as this is an internal step.